### PR TITLE
Fix broken links in Nomad post and JS pages

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #1052
- External link checking highlighted some broken links.
  - @avillela: please review the Nomad blog post updates. (Note: I ran Prettier over the file too.)
  - @open-telemetry/javascript-approvers: please review the link updates to the JS pages

Previews:

- https://deploy-preview-2355--opentelemetry.netlify.app/blog/2022/otel-demo-app-nomad/
- https://deploy-preview-2355--opentelemetry.netlify.app/docs/instrumentation/js/
- https://deploy-preview-2355--opentelemetry.netlify.app/docs/instrumentation/js/context/

### CODE REVIEW

File: content/en/docs/instrumentation/js/_index.md [hunk 1]
```diff
@@ -15,6 +15,6 @@ weight: 20
 ## Further Reading
 
 - [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
-- [Getting Started]({{< relref "getting-started" >}})
-- [API Reference](https://open-telemetry.github.io/opentelemetry-js-api)
+- [Getting Started](getting-started/)
+- [API Reference](https://open-telemetry.github.io/opentelemetry-js/modules/_opentelemetry_api.html)
```
Comment (chalin) [inline content/en/docs/instrumentation/js/_index.md:19]: @open-telemetry/javascript-approvers - is this a valid link for **API Reference** or should I just combine it with ** SDK Reference** as in:

> - [API and SDK reference](https://open-telemetry.github.io/opentelemetry-js)

Linking to https://open-telemetry.github.io/opentelemetry-js?

File: content/en/docs/instrumentation/js/context.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This document describes the OpenTelemetry context API for JavaScript and how it
 More information:
 
 - [Context specification](/docs/reference/specification/context/)
-- [Context API reference](https://open-telemetry.github.io/opentelemetry-js-api/classes/contextapi.html)
+- [Context API reference](https://open-telemetry.github.io/opentelemetry-js/classes/_opentelemetry_api.ContextAPI.html)
```
Comment (chalin) [inline content/en/docs/instrumentation/js/context.md:16]: @open-telemetry/javascript-approvers is this a valid updated link? If not, what should it be?

File: content/en/docs/instrumentation/js/_index.md (hunk 1 above)
Comment (cartermp) [reply content/en/docs/instrumentation/js/_index.md:19]: This should be https://open-telemetry.github.io/opentelemetry-js/

(it's the same link we use in the API node for the sidebar0

File: content/en/docs/instrumentation/js/context.md (hunk 2 above)
Comment (cartermp) [reply content/en/docs/instrumentation/js/context.md:16]: If we don't hear back soon-ish then I'd so let's go with this, it's correct insofar as I'm not aware of any other context API reference

File: content/en/docs/instrumentation/js/_index.md (hunk 1 above)
Comment (chalin) [reply content/en/docs/instrumentation/js/_index.md:19]: If we'll be using the same link then the two entries should be combined, as I suggested above. WDYT?

Comment (cartermp) [reply content/en/docs/instrumentation/js/_index.md:19]: Yeah, let's just combine it.

There's a _whole thing_ I have in my head about how there's a full API reference for the thing you download and install (the SDK, which includes the API), and the API reference for the "api", which is literally just the opentelemetry api itself, the thing designed to work with any arbitrary instantiated SDK. It's the right technical design but being this precise for end-users who primarily install an SDK anyways just makes things confusing.

**Comment by chalin** [comment]
@cartermp - do you think that you could review the blog post in case @avillela isn't available? Thanks!

**Comment by cartermp** [comment]
The post looks good to me.

**Review by cartermp — APPROVED** [review]

**Pull request merged**